browser/src/slideshow/SlideShowPresenter.ts

```diff
@@ -119,16 +120,18 @@ class SlideShowPresenter {
 	private _slideShowHandler: SlideShowHandler;
 	private _slideShowNavigator: SlideShowNavigator;
 	private _metaPresentation: MetaPresentation;
-	private _startSlide: number;
 	private _presentationInfoChanged: boolean = false;
 
-	constructor(map: any) {
+        constructor(map: any, options: any) {
 		this._map = map;
 		this._init();
+		if (options && options.noHooks)
+			return;
 		this.addHooks();
 	}
 
-	addHooks() {
+        addHooks() {
+	        this._map.on('presentationinfo', this.onSlideShowInfo, this);
 		this._map.on('newfullscreen', this._onStart, this);
```

Comment:
So, add ` this._map.off('presentationinfo', this.onSlideShowInfo, this)` in `removeHooks`.

Reply:
Thanks, I forgot it.